packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -225,77 +223,6 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
         });
     }
 
-    public async read(blobId: string): Promise<string> {
-        return this.readWithEncodingOutput(blobId, "base64");
-    }
-
-    /**
-     * {@inheritDoc @fluidframework/driver-definitions#IDocumentStorageService.readString}
-     */
-    public async readString(blobId: string): Promise<string> {
-        return this.readWithEncodingOutput(blobId, "string");
-    }
-
-    private async readWithEncodingOutput(blobId: string, outputFormat: string): Promise<string> {
```

Comment:
Some of this logic needs to be moved to readBlob, not just deleted.
Key changes required there:
1. Use cache (this.blobCache, this.scheduleClearBlobsCache logic from here)
2. Patching of attributes blob (though I hope we do not need to do so and @chumppchief can simply remove support of branches from here)